docs/csharp/language-reference/builtin-types/bool.md

```diff
@@ -0,0 +1,44 @@
+---
+title: "bool type - C# reference"
+ms.date: 11/26/2019
+f1_keywords: 
+  - "bool"
+  - "bool_CSharpKeyword"
+helpviewer_keywords: 
+  - "bool data type [C#]"
+  - "Boolean [C#]"
+ms.assetid: 551cfe35-2632-4343-af49-33ad12da08e2
+---
+# bool (C# reference)
+
+The `bool` type keyword is an alias for the .NET <xref:System.Boolean?displayProperty=nameWithType> structure type that represents a Boolean value, which can be either `true` or `false`.
+
+To perform logical operations with values of the `bool` type, use [Boolean logical](../operators/boolean-logical-operators.md) operators. The `bool` type is the result type of [comparison](../operators/comparison-operators.md) and [equality](../operators/equality-operators.md) operators. A `bool` expression can be a controlling conditional expression in the [if](../keywords/if-else.md), [do](../keywords/do.md), [while](../keywords/while.md), and [for](../keywords/for.md) statements and in the [conditional operator `?:`](../operators/conditional-operator.md).
+
+The default value of the `bool` type is `false`.
+
+## Literals
+
+You can use the `true` and `false` literals to initialize a `bool` variable or to pass a `bool` value:
+
+[!code-csharp-interactive[bool literals](~/samples/csharp/language-reference/builtin-types/BoolType.cs#Literals)]
+
+## Conversions
+
+C# doesn't provide built-in conversions between the `bool` type and other types. However, .NET provides the methods that you can use to convert to or from the `bool` type. For more information, see the [Converting to and from Boolean values](/dotnet/api/system.boolean#converting-to-and-from-boolean-values) section of the <xref:System.Boolean?displayProperty=nameWithType> API reference page.
```

Comment:
There's an inaccuracy here that I'm not sure how to phrase. There is an implicit identity nullable conversion from `bool` to `bool?`.  That's what's talked in the next section. Here's one suggestion, but I'm not thrilled with it.

```suggestion
The only built-in conversion involving `bool` is the implicit conversion from `bool` to `bool?`, discussed in the following section. .NET provides the methods that you can use to convert to or from the `bool` type. For more information, see the [Converting to and from Boolean values](/dotnet/api/system.boolean#converting-to-and-from-boolean-values) section of the <xref:System.Boolean?displayProperty=nameWithType> API reference page.
```

Reply:
Indeed, and then also explicit conversion from `bool?` back. I'm thinking about wording

Reply:
@BillWagner please take a look at the latest commit in which I've corrected the inaccuracy.